src/test/java/EdsonAnawayaTests/Entity/TestStr.java

```diff
@@ -0,0 +1,47 @@
+package EdsonAnawayaTests.Entity;
+
+import io.restassured.RestAssured;
+import io.restassured.builder.RequestSpecBuilder;
+import io.restassured.http.ContentType;
+import io.restassured.response.Response;
+import io.restassured.specification.RequestSpecification;
+import io.restassured.filter.log.RequestLoggingFilter;
+import io.restassured.filter.log.ResponseLoggingFilter;
+
+public class TestStr {
+    private static String token = "156721c42720001c6856e1afe292880e776ec1d5";
+    private static final String BASE_URL = "https://api.todoist.com";
+    private static RequestSpecification request;
+    private static Response response;
+
+    public static void authorization() {
+        request = RestAssured.given().auth().oauth2(token);
+    }
+
+    public static void configuration(String baseURI, String basePath) {
```

Comment:
you are receving the baseuri and basepath as argument but you have those valued as static strings